### ISSUE TITLE
Firefox WebRTC Fails to Connect Using Coturn Docker

### ISSUE THREAD

**Issue by Miko4699** [coturn/coturn#1619]
### Description
I am running Coturn in Docker using the image ghcr.io/coturn/coturn:4.6.3. Below is the Docker configuration I am using:

```
ports:
  - 3478:3478
  - 3478:3478/udp
  - 5349:5349
  - 5349:5349/udp
  - 49160-49180:49160-49180/udp
command:
  - "--listening-port=3478"
  - "--external-ip=$TURN_SERVER_IP"
  - "--min-port=49160"
  - "--max-port=49200"
  - "--static-auth-secret=$TURN_SERVER_SECRET"
  - "--realm=$TURN_SERVER_IP"
  - "--no-rfc5780"
  - "--response-origin-only-with-rfc5780"
  - "--fingerprint"
  - "--lt-cred-mech"
  - "--log-file=/var/log/turnserver/turnserver.log"
  - "--verbose"
```

However, Firefox cannot establish a WebRTC connection. The ICE candidates either fail or remain in progress：

![image](https://github.com/user-attachments/assets/81130c44-d09a-4fdc-bb57-262da0663176)

### Environment:
Coturn Docker Image: ghcr.io/coturn/coturn:4.6.3
Firefox：133.0.3 (64 bit）

**Comment by Miko4699** [comment]
Based on the Firefox，I used [icetestinfo ](https://icetest.info/) to test the turnserver username and password from the webrtc connection I just established

![image](https://github.com/user-attachments/assets/869c687c-e469-47b2-b0d4-8a4bdc11d7ee)

**Comment by ggarber** [comment]
does the same work in chrome?

**Comment by Miko4699** [comment]
1. Yes, the same configuration works perfectly fine in Chrome and connects successfully.

2. Additionally, I found that with Firefox versions 113 and later, if I force it to use only relay candidates, it can connect successfully.

**Comment by Miko4699** [comment]
> does the same work in chrome?

Yes, the same configuration works perfectly fine in Chrome and connects successfully.

Additionally, I found that with Firefox versions 113 and later, if I force it to use only relay candidates, it can connect successfully.

**Comment by ggarber** [comment]
can you share your server hostname for me to test it?  even without credentilas it should be able to gather a srvflx candidate with icetestinfo.

also, you can check Firefox ICE logs on the bottom of "about:webrtc" page in Firefox to see if it gives you any clue.

**Comment by gb3uuu** [comment]
I have the same configurations as @Miko4699 . I'm using coturn 4.7.0-r1-alpine on docker. I don't have any problems with any browser except Firefox. I can't connect in Firefox.
